packages/tools/src/tools/annotation/planarFreehandROITool/drawLoop.ts

```diff
@@ -185,10 +189,14 @@ function completeDrawClosedContour(element: HTMLDivElement): boolean {
   // Remove last point which will be a duplicate now.
   canvasPoints.pop();
 
+  const updatedPoints = assertInterpolation(this.configuration)
```

Comment:
Can you re-use the function from the previous request to figure out which set of points to get?  Eg the getInterpolatedOr... function?